release-notes/v1.2.0-rc.1.yaml

```diff
@@ -0,0 +1,131 @@
+  Added support for Datadog tracing support in EnvoyProxy CRD
+  Added support for request response sizes stats in EnvoyProxy CRD
+  Added support for wildcard matching for CORS AllowMethods and AllowHeaders settings in SecurityPolicy CRD
+  Added support for match conditions for access log in EnvoyProxy CRD
+  Added support for using BackendCluster to represent OIDCProvider
+  Added support for RecomputeRoute for ExtAuth in SecurityPolicy CRD
+  Added support for sharing token cookies between multiple domains in SecurityPolicy CRD
+  Added support for JSONPatches for proxy bootstrap modifications in EnvoyProxy CRD
+  Added support for LB priority for non xRoute endpoints
+  Added support for configuring the GRPC Health Checker in the BackendTrafficPolicy CRD
+  Added support for early request header mutation in the ClientTrafficPolicy CRD
+  Added support for JsonPath in the EnvoyPatchPolicy CRD
+  Added support for cluster settings for tracing and access log backends in EnvoyProxy CRD
+  Added support for cluster settings for non xRoute-generated backend refs
+  Added support for socket buffer limit field in ClientTrafficPolicy and BackendTrafficPolicy CRD
+  Added support for http2 upstream settings in BackendTrafficPolicy CRD
+  Added support for DNS resolution settings in BackendTrafficPolicy CRD
+  Added support for configuring service annotations in the Envoy Gateway helm chart
+  Added support for configuring priorityClassName to Envoy Gateway helm chart
+  Added support for ratelimit metrics monitoring in grafana in the addons helm chart
+  Added support for default user group and user id for the SecurityContexts in the Envoy Gateway helm chart
+  Added support for maxUnavailable in the PodDisruptionBudget in the Envoy Gateway helm chart
+  Added support for configuring NodeSelector in the Envoy Gateway helm chart
+  Added support for nonce in the OIDC auth flow
+  Added support for choosing an HTTPRoute's non-wildcard hostname as the default Host
+  Added support for returning 500 when EnvoyExtensionTrafficPolicy translation fails
+  Added support for returning 500 when SecurityPolicy translation fails
+  Added support for multiple backendRefs for ExtAuth and ExtProc
+  Added support for session persistence in HTTPRoute rules
+  Added support for the Backend resource for ExtAuth
+  Added support for target selectors on Envoy Gateway Extension Server policies
+  Added support for non-Kubernetes Backends for TLSRoute
+  Added support for fallback to the Backend API
+  Added support for reloadable EnvoyGateway configuration
+  Added support for adding Labels to the Envoy Service
+  Added support for custom name for ratelimit deployment
+  Added default SecurityContext for EG components
+  Added startupProbe to all provisioned containers
+  Added support for local validations for egctl translate and file provider
+  Added support for egctl x collect to collect information from the cluster for debugging
+
```

Comment:
* added support for a native prometheus metrics endpoint in the ratelimit server